src/main/java/main/Waypoints/WaypointManager.java

```diff
@@ -61,6 +65,7 @@ public static void transfer(){
                     PortSender.send(dataToSend);
                 }
             } else {
+            	GUI.error(model);
```

Comment:
this line could be removed, It just tells you the name of the aircraft you are using